html/aboutUs.html

```diff
@@ -0,0 +1,199 @@
+                        <a href="" class="linkedin ln-member3"><img src="https://www.oseyo.co.uk/wp-content/uploads/2020/04/linkedin-icon-png-circle-2.png" alt=""></a>
+                    </div>
+                </div>
+            </div>
+            <div class="member4 member-ctn">
+                <img src="https://tuwebsoluciones.com/wp-content/uploads/2018/03/team-member-4.jpg" alt="team-member" class="photo-member4 member-photo">
+                <div class="member-info">
+                    <p class="name-member4 name-member">Juan José</p>
+                    <p class="position-member4">Desarrollador Backend</p>
+                    <div class="social-media">
+                        <a href="" class="twitter tw-member4"><img src="https://cdn4.iconfinder.com/data/icons/social-media-icons-the-circle-set/48/twitter_circle-512.png" alt=""></a>
+                        <a href="" class="linkedin ln-member4"><img src="https://www.oseyo.co.uk/wp-content/uploads/2020/04/linkedin-icon-png-circle-2.png" alt=""></a>
+                    </div>
+                </div>
+            </div>
+            <div class="member5 member-ctn">
+                <img src="https://tuwebsoluciones.com/wp-content/uploads/2018/03/team-member-4.jpg" alt="team-member" class="photo-member4 member-photo">
+                <div class="member-info">
+                    <p class="name-member5 name-member">Alejandro</p>
+                    <p class="position-member5">Desarrollador Frontend</p>
+                    <div class="social-media">
+                        <a href="" class="twitter tw-member5"><img src="https://cdn4.iconfinder.com/data/icons/social-media-icons-the-circle-set/48/twitter_circle-512.png" alt=""></a>
+                        <a href="" class="linkedin ln-member5"><img src="https://www.oseyo.co.uk/wp-content/uploads/2020/04/linkedin-icon-png-circle-2.png" alt=""></a>
+                    </div>
+                </div>
+            </div>
+            <!-- <div class="member6 member-ctn">
+                <img src="https://tuwebsoluciones.com/wp-content/uploads/2018/03/team-member-4.jpg" alt="team-member" class="photo-member6 member-photo">
+                <div class="member-info">
+                    <p class="name-member6 name-member">Óscar</p>
+                    <p class="position-member6">Maquetador</p>
+                    <div class="social-media">
+                        <a href="" class="twitter tw-member6"><img src="https://cdn4.iconfinder.com/data/icons/social-media-icons-the-circle-set/48/twitter_circle-512.png" alt=""></a>
+                        <a href="" class="linkedin ln-member6"><img src="https://www.oseyo.co.uk/wp-content/uploads/2020/04/linkedin-icon-png-circle-2.png" alt=""></a>
+                    </div>
+                </div>
+            </div> -->
```

Comment:
Hola Aleja, ¿esta parte comentada sí va o no va?

Reply:
@Salacodigo it's possible that we need it